addons/udes_stock/models/stock_location.py

```diff
@@ -597,6 +606,19 @@ def get_inventory_adjustment_data(self):
             "location_id": self.id,
             "filter": "none",
             "state": "confirm",
+            "user_id": self.env.user.id,
+            "u_date_start_scanned": self.u_date_start_scanned,
+        }
+
+    def get_inventory_validated_data(self):
```

Comment:
This function is the same as the previous one besides the state, can we have a common function to get default data?
Otherwise, if we want to change the name  (`"PI inventory validated " + self.name,`) to something else we will have to change it twice or we might forget one of them.

Reply:
Yes i was aware tbh :), just thought it would look like more complex to send name as parameter :(